### ISSUE TITLE
Offered to select being a "new speaker" when updating my settings on the site.

### ISSUE THREAD

**Issue by sjn88** [dddeastanglia/DDDEastAnglia#30]
I may have registered for another reason, so I don't think this belongs here, but possible on the "new session" page?

**Comment by adrianbanks** [comment]
I'm working on these pages at the moment, so I'll take a look at this.

**Comment by adrianbanks** [comment]
I think this is correct where it is. At the moment, I believe the plan is for only speakers to need to register on the site. Voting won't require registration, and registration for the event will be done with an external provider (e.g. EventBrite).

**Comment by sjn88** [comment]
You could add something to imply registering is only for speakers?

**Comment by adrianbanks** [comment]
Closing this, as I don't think we should do anything. It would be odd to put it on the session page, as it is a setting that belongs to the speaker, hence it being on the profile page. 

We have changed the registration process so that after registering, you are prompted to update your profile, so it should be easier to find now.

I don't think we've discussed _who_ will register on the site, but I still think that it is only speakers who will need to.
